progression/src/main/java/me/mykindos/betterpvp/progression/tips/tips/FishingExperienceTip.java

```diff
@@ -0,0 +1,38 @@
+package me.mykindos.betterpvp.progression.tips.tips;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import me.mykindos.betterpvp.core.tips.types.IRunCommand;
+import me.mykindos.betterpvp.core.utilities.UtilMessage;
+import me.mykindos.betterpvp.progression.Progression;
+import me.mykindos.betterpvp.progression.profile.ProfessionProfile;
+import me.mykindos.betterpvp.progression.tips.ProgressionTip;
+import net.kyori.adventure.text.Component;
+import org.bukkit.entity.Player;
+
+@Singleton
+public class FishingExperienceTip extends ProgressionTip implements IRunCommand {
+
+    @Inject
+    public FishingExperienceTip(Progression progression) {
+        super(progression, 1, 1);
+        setComponent(getComponent());
+    }
+
+    @Override
+    public Component getComponent() {
+        Component runComponent = runCommand("/fishing");
+        return UtilMessage.deserialize("By Fishing, you earn fishing experience. As you gain experience " +
+                "you will level up. Everytime you level up, you will be able to allocate skill points in ").append(runComponent);
```

Comment:
Needlessly long imo

Level up your fishing by catching fish. You can spend your fishing skill points in /fishing

Same for woodcutting

Reply:
I'm a tad wordy at times